store.go

```diff
@@ -26,12 +26,20 @@ var (
 	ErrTraceNotFound = errors.New("trace not found")
 )
 
+type TracesOpts struct {
+	Timespan Timespan
+}
+
+func DefaultTracesOpts() TracesOpts {
+	return TracesOpts{}
+}
+
 // A Queryer indexes spans and makes them queryable.
 type Queryer interface {
 	// Traces returns an implementation-defined list of traces. It is
 	// a placeholder method that will be removed when other, more
 	// useful methods are added to Queryer.
-	Traces() ([]*Trace, error)
+	Traces(opts TracesOpts) ([]*Trace, error)
```

Comment:
Maybe it would be nicer to use `opts *TracesOpts` and then treat `nil` implicitly as `DefaultTracesOpts()`?